### ISSUE TITLE
Unable to upgrade build process to EAP IC-212.3116.43

### ISSUE THREAD

**Issue by ahus1** [asciidoctor/asciidoctor-intellij-plugin#781]
## Observed vs. expected behavior

When upgrading to EAP IC-212.3724.25, the build fails with an exception. 

The is a known problem upstream. It seems that some of the Jars in IntelliJ miss directory entries, therefore there is a mismatch between the META-INF/INDEX.LIST and the actual entries in the archive.

See https://youtrack.jetbrains.com/issue/IDEA-271075

## Steps to reproduce

* upgrade in build.gradle to IC-212.3116.43
* run unit test AsciiDocXrefWithNaturalCrossReferenceInspectionTest
* the test will fail with the exception listed below

```
Caused by: jdk.internal.util.jar.InvalidJarIndexError: Invalid index
	at java.base/jdk.internal.loader.URLClassPath$JarLoader.getResource(URLClassPath.java:1025)
	at java.base/jdk.internal.loader.URLClassPath$JarLoader.getResource(URLClassPath.java:937)
	at java.base/jdk.internal.loader.URLClassPath.getResource(URLClassPath.java:314)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:695)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:621)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:579)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:576)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:398)
	at org.jruby.javasupport.JavaSupportImpl.loadJavaClass(JavaSupportImpl.java:157)
	at org.jruby.javasupport.Java.getProxyClassOrNull(Java.java:961)
```

**Comment by ahus1** [comment]
As listed upstream, the first exception occurs with groovy-2.5.14.jar looking for "org/asciidoctor.class". 

Apparently between ideaIC-212.3116.43 and IC-212.3724.25 the JAR file started missing the directory entry "/org"

**Comment by ahus1** [comment]
Fixed in EAP IC-212.4321.30
